portable/NetworkInterface/ThirdParty/MSP432/NetworkInterface.c

```diff
@@ -827,6 +824,8 @@ BaseType_t xNetworkInterfaceOutput( NetworkBufferDescriptor_t * const pxDescript
 {
     struct NetworkInterfaceDataOut NIDataOutput;
     BaseType_t txResp;
+    BaseType_t returnValue;
```

Comment:
maintainability: prefer `BaseType_t returnValue = pdFALSE;` here.

Reply:
Yes, good idea.  This has been changed.